python/tvm/relay/backend/graph_runtime_codegen.py

```diff
@@ -57,17 +54,14 @@ def __init__(self, mod, target):
         self._setup(mod, target)
 
     def _setup(self, mod, target):
-        tgts = []
+        tgts = {}
         if isinstance(target, dict):
             for kv in target.items():
```

Comment:
```suggestion
            for device, target in target.items():
```

Reply:
This is from the old commit. But I can change it anyway.